ui/panels-plugin/src/plugins/time-series-chart/time-series-chart-model.ts

```diff
@@ -66,7 +66,9 @@ export const DEFAULT_PALETTE: PaletteOptions = {
   kind: 'Auto',
 };
 
-export const DEFAULT_POINT_RADIUS = 4;
+// How much larger datapoint symbols are than line width, also applied in VisualOptionsEditor.
+export const POINT_SIZE_OFFSET = 0.75;
```

Comment:
Planning to sync up with @foxbat07, I don't think `0.75` for the difference between symbol size and line width is enough, [Happo test example](https://happo.io/a/1009/p/1488/compare/a7da247a13940267c5ec532ad9e60726d0fe4b65/d90e895978ed47fe3bb142f739a7d2b1f7cb4f5c) really shows why it probably need to be slightly larger: 

![image](https://user-images.githubusercontent.com/9369625/231639868-f5a71585-5d60-4cb5-b657-ec98af7cc48b.png)

Reply:
Increased POINT_SIZE_OFFSET slightly from 0.75 to 1.5 based on feedback